python/cuml/linear_model/ridge.pyx

```diff
@@ -133,7 +131,7 @@ class Ridge:
 
     """
 
-    def __init__(self, alpha=1.0, solver='eig', fit_intercept=True, normalize=False):
```

Comment:
changing the `alpha` from `1.0` to `1` changes it from float to int, which could have unintended consequences, please change it back. 

Also `fit_intercept` and `normalize` has to be either `False` or `True`, setting it up as `None` will make the call fail with the default arguments. Why the change of solver?